src/main/kotlin/no/nav/syfo/senoppfolging/v2/SenOppfolgingVeilederADControllerV2.kt

```diff
@@ -27,7 +31,7 @@ class SenOppfolgingVeilederADControllerV2(
     val veilederTilgangClient: VeilederTilgangClient,
     val responseDao: ResponseDao,
 ) {
-    private val cutoffDate = LocalDate.now().minusMonths(3)
+    private val cutoffDate = LocalDate.of(2024, Month.JULY, 1)
```

Comment:
Hva er denne til egentlig?

Reply:
Trengs kun fordi `responseDao.findLatestFormResponse()` som brukes lenger ned her tar inn en from-dato. Kan evt lage en egen variant i `responseDao` som ikke krever dato så blir det litt mindre å lure på for fremtids-syfo :thinking: